Source/WebCore/platform/graphics/cocoa/MediaPlaybackTargetContext.h

```diff
@@ -57,21 +56,43 @@ class MediaPlaybackTargetContext {
     MediaPlaybackTargetContext() = default;
     WEBCORE_EXPORT explicit MediaPlaybackTargetContext(RetainPtr<AVOutputContext>&&);
 
-    MediaPlaybackTargetContext(RetainPtr<NSData>&& serializedOutputContext, bool hasActiveRoute)
-        : m_type(Type::SerializedAVOutputContext)
-        , m_serializedOutputContext(WTFMove(serializedOutputContext))
-        , m_cachedHasActiveRoute(hasActiveRoute)
-    {
-        ASSERT(m_serializedOutputContext);
-    }
-
     MediaPlaybackTargetContext(const String& mockDeviceName, MockState state)
         : m_type(Type::Mock)
         , m_mockDeviceName(mockDeviceName)
         , m_mockState(state)
     {
     }
 
+    struct NonPlatformData {
+        String mockDeviceName;
+        MockState mockState;
+    };
+
+    struct PlatformData {
```

Comment:
I don't think we need this struct, we can use `RetainPtr<AVOutputContext>` directly.